api/db-pouch.js

```diff
@@ -2,17 +2,26 @@ const PouchDB = require('pouchdb-core');
 PouchDB.plugin(require('pouchdb-adapter-http'));
 PouchDB.plugin(require('pouchdb-mapreduce'));
 
-const { COUCH_URL } = process.env;
-// strip trailing slash from to prevent bugs in path matching
-const couchUrl = COUCH_URL && COUCH_URL.replace(/\/$/, '');
+const { COUCH_URL, UNIT_TEST_ENV } = process.env;
 
-const db = {};
+if(UNIT_TEST_ENV) {
+  module.exports.medic = {
+    allDocs: () => Promise.resolve({ offset:0, total_rows:0, rows:[] }),
+    bulkDocs: () => Promise.resolve([]),
+    put: doc => Promise.resolve({ ok:true, id:doc._id, rev:'1' }),
+    query: () => Promise.resolve({ offset:0, total_rows:0, rows:[] }),
+  };
```

Comment:
Nah bro, this is real dodge. Tests will pass without mocking db calls which could cause them to pass incorrectly. The nano implementation just has null functions to avoid this.

I was really hoping we'd find a better way in the pouch util so we don't have to list out every property in the pouchdb api but I haven't had a chance to look at it yet.

One way would be to have a getter function which you could mock entirely but it makes the code a bit uglier, eg: `db.medic().put(...)` instead of `db.medic.put(...)`.

Alternatively, it looks like using sinon sandboxes we can stub properties properly! There's a [`sandbox.stub()`](http://sinonjs.org/releases/v1.17.7/sandbox/) function which we can call in our tests like...

```js
sandbox.stub(db, 'medic', { put: sinon.stub() });
```

Then a `sandbox.restore()` should revert medic back to what it was before. Wanna give it a try?